contributing-guides/translation-templates/alias-pages.md

```diff
@@ -34,6 +34,7 @@ The templates can be changed when necessary.
 [uk](#uk) •
 [zh](#zh) •
 [zh_TW](#zh_tw)
+[ar](#ar) •
```

Comment:
Please keep the alphabetic order, `ar` should be before `bs`.

Reply:
and `en` too?

Reply:
`en` is an exception, since it's the default language